Program_1.0.py

```diff
@@ -72,3 +76,6 @@ def phrase_with_tag(country_name, tag):
         data = phrase_with_tag(country_name, tag)
         for phrase in data:
             print(phrase)
+elif country_name == '' and tag != '':
```

Comment:
#pierdola ale nie potrzbujesz tego :D

Reply:
Zostawiłem to, bo będzie jeszcze else

Reply:
ideeowo jak robisz PR to maja tu byc TYLKO rzeczy ktore dotycza danego zadania i zero niepotrzebnego kodu :P w tym wypadku to jest niepotrzebny kod bo nic nie robi ale to tylko pierdola takze mozesz mergowac